sdk/storage/azure-storage-blob/azure/storage/blob/_upload_helpers.py

```diff
@@ -133,10 +139,10 @@ def upload_block_blob(  # pylint: disable=too-many-locals, too-many-statements
             total_size = length
             encryptor, padder = None, None
             if encryption_options and encryption_options.get('key'):
-                cek, iv, encryption_data = generate_blob_encryption_data(
+                cek, iv, encrypted_data = generate_blob_encryption_data(
```

Comment:
nit: Could we use a different name than `encrypted_data`. That makes it sound like the variable holds actual data, when it's really just metadata. Maybe `encryption_metadata`?

P.S. I really don't like that MyPy is complaining about this. The first variable is out of scope here so we should be able to use it again (even though scope does not work like that in Python)